C/primitive/elements.c

```diff
@@ -441,9 +506,10 @@ extern transaction* elements_simplicity_mallocTransaction(const rawTransaction*
  *               NULL != amr implies unsigned char amr[32]
  *               NULL != file;
  */
-extern bool elements_simplicity_execSimplicity(bool* success, unsigned char* imr, const transaction* tx, uint_fast32_t ix,
-                                               const unsigned char* cmr, const unsigned char* amr, FILE* file) {
-  if (!success || !tx || !file) return false;
+extern bool elements_simplicity_execSimplicity( bool* success, unsigned char* imr
+                                              , const transaction* tx, uint_fast32_t ix, const tapEnv* taproot
+                                              , const unsigned char* cmr, const unsigned char* amr, FILE* file) {
+  if (!success || !tx || !taproot || !file) return false;
```

Comment:
The precondition can be updated to reflect NULL != taproot

Reply:
Fixed.